easybuild/easyconfigs/j/JasPer/JasPer-1.900.1-CrayGNU-18.08.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'JasPer'
+version = '1.900.1'
+
+homepage = 'http://www.ece.uvic.ca/~frodo/jasper/'
+description = "gzip (GNU zip) is a popular data compression program as a replacement for compress"
```

Comment:
wrong description ?

Reply:
It was actually wrong in most of the existing JasPer recipies. So, I changed it in all.